workshop/content/my-first-enclave/nitro-enclaves-cli/_index.en.md

```diff
@@ -51,15 +51,20 @@ For detailed steps on installing the CLI, see [Prerequisites and environment set
     </pre>
 
     {{% notice info %}}
-The default values are `512MB` and `2 CPUs`. You can update these values and should be less than RAM and CPU available on EC2 instance. The changes require restarting `nitro-enclaves-allocator` service.
+The default values are `512MB` and `2 vCPUs`. You can update these values and should be less than free memory and number of cores available on EC2 instance. 
     {{% /notice %}}
 
 1. Check available RAM and vCPU on EC2 instance. `CPU` and `RAM` reserved for the enclave should be less than the free available `RAM` and `CPU` on the parent EC2 instance.
     ```sh
-    $ free -m
+    $ free -h
```

Comment:
This was originally `-m` to match the MiB that's used in the allocator yaml config